pkg/build/api/types.go

```diff
@@ -19,6 +19,12 @@ const (
 	BuildCloneAnnotation = "openshift.io/build.clone-of"
 	// BuildPodNameAnnotation is an annotation whose value is the name of the pod running this build
 	BuildPodNameAnnotation = "openshift.io/build.pod-name"
+	// BuildJenkinsStatusJSONAnnotation is an annotation holding the Jenkins status information
+	BuildJenkinsStatusJSONAnnotation = "openshift.io/jenkins-status-json"
```

Comment:
@mfojtik I don't see we're at liberty to choose the name here - it's already chosen.  Or is the request to go and change it in the UI & Jenkins plugin?